lib/logging.rb

```diff
@@ -0,0 +1,30 @@
+class Logging
```

Comment:
I like the way we've extracted this! It's not clear that it's not a built-in Rails class. Could we put a namespace on it, perhaps?

More minor: this is acting more like a module than a class — could it be one?

Reply:
I have changed the name from Logging to LogstashLogging.